### PR TITLE
add callout for collections synching

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr turns the existing content about the once a day sync into a callout and adds info on how to trigger a manual sync per [slack thread](https://dbt-labs.slack.com/archives/C06GJRK27PF/p1733359577763749) and user confusion.

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-mirnawong1-patch-26-dbt-labs.vercel.app/docs/cloud-integrations/configure-auto-exposures

### CODE REVIEW

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 1]
```diff
@@ -61,9 +61,15 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
 4. Select the collections you want to include for auto exposures. 
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md:65]:
```suggestion

#### Synch collections
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 2]
```diff
@@ -61,9 +61,16 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
...
+#### Synch collections
+   :::info Syncing collections
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]:
```suggestion
   :::info
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 3]
```diff
@@ -61,9 +61,16 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
 4. Select the collections you want to include for auto exposures. 
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+#### Synch collections
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]:
```suggestion
#### Sync collections
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 4]
```diff
@@ -61,9 +61,16 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
 4. Select the collections you want to include for auto exposures. 
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+#### Sync collections
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]:
```suggestion
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 5]
```diff
@@ -61,9 +61,16 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
...
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+   #### Sync collections
+      :::info
+      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
+   
+      To manually trigger a sync and update collections (including additions or deletions), go to **Collections**, click **Edit**, make your selections or deselections, and then click **Save**.
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]:
```suggestion
      To manually trigger a sync and update collections (including additions or deletions), go to **Collections**, click **Edit**, make your checkbox selections or deselections, and then click **Save**. You should see the updates in Explorer.
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 6]
```diff
@@ -61,9 +61,16 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
...
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+   #### Sync collections
+      :::info
+      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
+   
+      To manually trigger a sync and update collections (including additions or deletions), go to **Collections**, click **Edit**, make your checkbox selections or deselections, and then click **Save**. You should see the updates in Explorer.
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]: @joellabes @eddowh - do they have to refresh their browser in explorer or will it just show up?

Comment (joellabes) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]:
```suggestion
      A sync will also begin immediately when the list of selected collections is modified, to capture any newly-selected workbooks and remove those which are no longer relevant.
```

**Review by joellabes — DISMISSED** [review]

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 7]
```diff
@@ -61,9 +61,16 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
 4. Select the collections you want to include for auto exposures. 
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+   #### Sync collections
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]:
```suggestion
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 8]
```diff
@@ -61,9 +61,15 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
...
 4. Select the collections you want to include for auto exposures. 
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+      :::info
+      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
+   
+      A sync will also begin immediately when the list of selected collections is modified, to capture any newly-selected workbooks and remove those which are no longer relevant.
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]: making it clear its the list in cloud
```suggestion
      dbt Cloud immediately starts a sync when you update the selected collections list, capturing new workbooks and removing irrelevant ones.
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 9]
```diff
@@ -61,9 +61,15 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
...
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+      :::info
+      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
+   
+      dbt Cloud immediately starts a sync when you update the selected collections list, capturing new workbooks and removing irrelevant ones.
+      :::
+
+6. Click **Save**.
```
Comment (matthewshaver) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]: Shouldn't this still be 5?

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 10]
```diff
@@ -61,9 +61,15 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
 4. Select the collections you want to include for auto exposures. 
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+      :::info
+      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
```
Comment (matthewshaver) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]: With what you've added below, does this make more sense?
```suggestion
      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next sync (automatically once per day).
```

**Comment by mirnawong1** [comment]
merging this pr as approved by joel and folded in Matt's changes

**Pull request merged**